extensions/guacamole-auth-radius/src/licenses/bundled/README

```diff
@@ -0,0 +1,4 @@
+Apache Guacamole includes a number of subcomponents with separate copyright
+notices and license terms. Your use of these subcomponents is subject to the
```

Comment:
Sounds like there are a couple of potential routes here:
- Go with the current roadmap and just distribute the source.
- Build/distribute a binary that includes everything except the Category-X licenses, with instructions that these additional dependencies need to be retrieved/downloaded.  I don't know how feasible this is with maven - is it possible to tell it there are dependencies, but *not* to package them?  Also, if we went this route, is there a place where it would make sense to tell users to drop the JAR file(s) for those dependencies?  And is this really any simpler than just telling folks to download a Java compiler and Maven and build it on their own?